pom.xml

```diff
@@ -93,6 +93,7 @@ under the License.
 		<flink.forkCount>1C</flink.forkCount>
 		<flink.reuseForks>true</flink.reuseForks>
 		<log4j.configuration>log4j-test.properties</log4j.configuration>
+		<argLine>-Xms256m -Xmx800m -XX:-UseGCOverheadLimit</argLine>
```

Comment:
I would prefer if the argline configuration stayed as is. It is missing the fork number here. This requires us to manually insert it for all modules which rely on it.